### ISSUE TITLE
Query with timeDimension "granularity: minute" and "dateRange: Today" produces bad SQL-query

### ISSUE THREAD

**Issue by flo-ff** [cube-js/cube.js#2688]
**Describe the bug**
When defining a measure with a rollingWindow, scoping the granularity to "minute" and dateRange to "Today" procues a query with date values that consists of a lot of duplicates and are incomplete;

**Measure**:
```
measures: {
    enters: {
      sql: `enters`,
      type: `sum`,
      rollingWindow: {
        trailing: `1 day`,
        offset: "start"
      }
    },
  },
```

produces **SQL-query**

```
SELECT
  q_0."frequency__created_minute",
  "frequency__enters" "frequency__enters"
FROM
  (
    SELECT
      "Frequency.created_series"."date_from" "frequency__created_minute",
      sum("frequency__enters") "frequency__enters"
    FROM
      (
        SELECT
          date_from:: timestamp,
          date_to:: timestamp
        FROM
          (
            VALUES
              (
                '2021-05-10T00:05:00.000',
                '2021-05-10T00:05:59.999'
              ),
              (
                '2021-05-10T00:05:00.000',
                '2021-05-10T00:05:59.999'
              ),
              (
                '2021-05-10T00:05:00.000',
                '2021-05-10T00:05:59.999'
              ),
              (
                '2021-05-10T00:05:00.000',
                '2021-05-10T00:05:59.999'
              ),
...
              (
                '2021-05-10T01:05:00.000',
                '2021-05-10T01:05:59.999'
              ),
              (
                '2021-05-10T01:05:00.000',
                '2021-05-10T01:05:59.999'
              ),
              (
                '2021-05-10T01:05:00.000',
                '2021-05-10T01:05:59.999'
              ),
              (
                '2021-05-10T01:05:00.000',
                '2021-05-10T01:05:59.999'
              ),
```
As you can see, the values which should cover every minute-interval inside the hour only cover the 5-6 minute interval and none else. I expect to see

```
        FROM
          (
            VALUES
              (
                '2021-05-10T00:00:00.000',
                '2021-05-10T00:00:59.999'
              ),
              (
                '2021-05-10T00:01:00.000',
                '2021-05-10T00:01:59.999'
              ),
              (
                '2021-05-10T00:02:00.000',
                '2021-05-10T00:02:59.999'
              ),
              (
                '2021-05-10T00:03:00.000',
                '2021-05-10T00:03:59.999'
              ),
...
```

Having "granularity: hour" works just fine and produces correct result.

**To Reproduce**
1. Define measure with rollingWindow
2. Define time dimension
3. Go to playground
4. In playground, select measure from 1.
5. In playground, as time-dimension select from 2.
6. In playground, as "for" select "Today"
7. In playground, as "by" select "Minute"
8. In playground, check generated SQL

**Expected behavior**
Values should be generated correctly

**Screenshots**
![image](https://user-images.githubusercontent.com/12515931/117675396-cf93e700-b1ac-11eb-9669-af9e3aa9c8a3.png)

**Version:**
cubejs/cube:v0.27.7-alpine

**Comment by hassankhan** [comment]
Hi @flo-ff :wave:

Thanks for posting this issue and detailed reproduction steps 👍  Just wondering which database driver you're using in the above example

**Comment by flo-ff** [comment]
Hi @hassankhan,

I'm using **Postgres**.

Cheers

**Comment by ovr** [comment]
Hello @rongfengliang ,

Thank you for submitting the issue. Fixed & released in `v0.27.14`

Thanks
